app/browser/api/ledgerNotifications.js

```diff
@@ -43,6 +44,15 @@ const sufficientBalanceToReconcile = (state) => {
   const budget = ledgerState.getContributionAmount(state)
   return balance + unconfirmed >= budget
 }
+const noBackupKeys = (state) => {
+  return !state.getIn(['about', 'preferences', 'backupSucceeded']) && !state.getIn(['about', 'preferences', 'recoverySucceeded'])
```

Comment:
lets create new state helper `aboutPreferences` in `app/common/state`. with three functions `hasBeenBackedUp` (or something like this instead of `noBackupKeys`, `getProp` and `setProp`.